rc/bin/cpurc

```diff
@@ -12,6 +12,15 @@ sysname=`{cat /dev/sysname}
 # parallelism for mk
 NPROC = `{wc -l </dev/sysstat}
 
+# set up any partitions
+diskparts
+
+# start up local swapping
```

Comment:
no. take this out.

Reply:
it will fail anyway, just don't create swap partitions, but ... it has not worked in years.